_data/versions.yml

```diff
@@ -1346,6 +1346,102 @@ v2.1:
       version: 1.4.1
       url: http://git.openstack.org/cgit/openstack/networking-calico/commit/?h=1.4.1
 
+v3.0:
+- title: v3.0.0-beta1
+  note: |
+    17 November 2017
+    
+    This is a pre-release of v3.0.0. This release is intended for testing purposes only and is NOT to be used on production systems.
+
+    #### What's new
+
+    - Support for the [etcd version 3 datastore](https://coreos.com/blog/etcd3-a-new-etcd.html).
+
+    - Two new `calicoctl` resources: [BGP Configuration](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/bgpconfig) and [Felix Configuration](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/felixconfig).
+
+    - Those using the Kubernetes API datastore can now use `calicoctl` to create, read, update, and delete {{site.prodname}} policies. 
+    
+    - The `calicoctl` policy resource has been split into new [network policy](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/networkpolicy) and [global network policy](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/globalnetworkpolicy) resources.
+    
+    - [Network policy](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/networkpolicy) resources can include a `namespace` value, allowing you to create policies that only apply to [workload endpoint resources](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/workloadendpoint) in the same namespace.
+    
+    - You can now create `namespaceSelector` expressions in [network policy](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/networkpolicy) and [global network policy](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/globalnetworkpolicy) resources to apply a policy to an entire namespace.
```

Comment:
This isn't _quite_ right. How about:

```
You can now specify `namespaceSelector` expressions in network policy and global network policy rules to select one or more namespaces in their entirety. 
```

Main differences are:
- create -> specify, since a `namespaceSelector` can't really be created in the sense it's typically used for Calico objects.
- "apply to an entire namespace" isn't quite accurate for two reasons:
  - it's only in rules, so it can perform actions on entire namespaces, but can't apply a single policy to multiple namespaces this way.
  - This implies it can only select a single namespace, when in fact it selects _multiple_ namespaces.

Reply:
OK good to know, thanks for the clarification!